src/tools/FirmwareUpdater/FirmwareUpdater.pro

```diff
@@ -0,0 +1,114 @@
+#-------------------------------------------------
```

Comment:
.pro file is usefull because you can still open the project with QtCreator. It is not used in other contexts.